src/util/RouterUtil.js

```diff
@@ -133,17 +133,12 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
 
       var redirectFn = router.settings.get('redirectUtilFn');
 
-      if (res.type === Enums.SESSION_STEP_UP) {
-        var targetUrl = res._links && res._links.next && res._links.next.href;
-        successData.stepUp = {
```

Comment:
Also strictly speaking this is breaking change (`successData` which is parameter to global success function).
Although we're fine with breaking change, is it better to add deprecate notes/logger instead of removing until 3.0?

Basically add `successData.next` and keep `stepUp` logic unchanged until 3.0 and go ahead to use `successData.next` from monolith.

thoughts?

Reply:
Good point here 👍 , it is a breaking change. I can keep the `stepUp` logic and remove it when we go to 3.0 as you said. Wonder if we can already modify the `README.md` (https://github.com/okta/okta-signin-widget/blame/master/README.md#L247) to encourage to use the new way (but as you said keep the old way working) or if we should wait 3.0 also for that, what's your opinion @haishengwu-okta @nbarbettini?

Reply:
sounds good idea to update README

Reply:
Thinking more about `README.md` changes (thanks @robertdamphousse-okta!) we will have to wait to update it, since old behavior (`stepUp` format) still has priority on new behavior (original/next format) and new APIs format that returns original link are still under FF/not ready, so developers cannot really use the new way yet.